### PR TITLE
feat: go import handler based on treesitter

### PR DESCRIPTION
**Pull request by sportshead**
Uses treesitter in order to find a Go `import-spec`, then opens https://pkg.go.dev/ for the package.

e.g.:
```go
import "fmt" // https://pkg.go.dev/fmt

import (
    "github.com/joho/godotenv" // https://pkg.go.dev/github.com/joho/godotenv
    git "github.com/jesseduffield/lazygit" // https://pkg.go.dev/github.com/jesseduffield/lazygit
)
```
Note the last example: using `gx` while focused anywhere from the `git` alias to the end of the path string opens the correct URL, since the handler searches for the parent `import-spec` statement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,21 +6,22 @@
 
 ## ✨ Features
 
-* open links without `netrw`
```
Comment (chrishrb) [inline README.md:9]: why did you change these all to hyphens ?

**Review by chrishrb — CHANGES_REQUESTED** [review]
The handler works as expected. 👍  Can you please add some tests and maybe change the hyphens in the readme back to asterisks?

File: README.md (hunk 1 above)
Comment (sportshead) [reply README.md:9]: sorry about that, seems like my LSP formatted the markdown file

I'll revert the changes and fix

**Review by chrishrb — APPROVED** [review]
